gateway/README.md

```diff
@@ -45,7 +45,9 @@ See the [openfaas/store](https://github.com/openfaas/store) repo for more.
 
 ## Logs
 
-Logs are available at the function level and can be accessed through Swarm or Kubernetes using native tooling. You can also install a Docker logging driver to aggregate your logs. By default functions will not write the request and response bodies to stdout. You can toggle this behaviour by setting `read_debug` for the request and `write_debug` for the response.
```

Comment:
The note on a logging aggregation tool is still valid, I'm not sure that should be deleted.